core/embed/extmod/modtrezorutils/modtrezorutils.c

```diff
@@ -257,6 +288,12 @@ STATIC const mp_rom_map_elem_t mp_module_trezorutils_globals_table[] = {
      MP_ROM_PTR(&mod_trezorutils_reboot_to_bootloader_obj)},
     {MP_ROM_QSTR(MP_QSTR_usb_data_connected),
      MP_ROM_PTR(&mod_trezorutils_usb_data_connected_obj)},
+    {MP_ROM_QSTR(MP_QSTR_device_variant_present),
+     MP_ROM_PTR(&mod_trezorutils_device_variant_present_obj)},
+    {MP_ROM_QSTR(MP_QSTR_device_color),
+     MP_ROM_PTR(&mod_trezorutils_device_color_obj)},
+    {MP_ROM_QSTR(MP_QSTR_device_btconly),
+     MP_ROM_PTR(&mod_trezorutils_device_btconly_obj)},
```

Comment:
What do you think of dropping `device_variant_present` and instead doing
```
def device_color() -> int | None:
def device_btconly() -> bool | None:
```

Reply:
https://github.com/trezor/trezor-firmware/pull/2961/commits/b1bcaab232c2776b142d3bdda7b5e58215e71029